gui/slick/js/core.js

```diff
@@ -2643,11 +2655,13 @@ var SICKRAGE = {
             // Moved and rewritten this from displayShow. This changes the button when clicked for collapsing/expanding the
             // "Show History" button to show or hide the snatch/download/failed history for a manual searched episode or pack.
             $(function() {
-                $('.collapse.toggle').on('hide.bs.collapse', function () {
+                $('body').on('hide.bs.collapse', '.collapse.toggle', function () {
```

Comment:
I started with binding these to 'body', but maybe that wasn't needed in hindsight.